client/src/constants/gtm-tags.js

```diff
@@ -86,8 +86,18 @@ module.exports = {
     cancelVerify: "_CV",
     accountVerified: "_AV",
     checkBadge: "_AV_CB",
-    requests: "_RQS",
-    offers: "_OFS",
+    requests: "_RQ",
```

Comment:
change this to _RE

Reply:
Will removing RQS, OFS from this file affect existing GTM tags that might be using them?